app/views/users/show.html.erb

```diff
@@ -1,9 +1,26 @@
 <section class="user-section">
-  <h2><%= "Name: #{@user.name}" %></h2>
+  <div class="profile-name">
+    <div class="profile">
+      <%= gravatar_for current_user, size: 100 %> 
+        <h2><%= "#{@user.name}" %></h2>
+    </div>
+    <div>
+      <% if request_sent?(@user.id) == 'pending' %>
+        <button class="friend_request">Pending</button>
+      <% elsif request_sent?(@user.id) == 'rejected' %>
+        <button class="friend_request">
+          <%= link_to 'Invite to friendship',  send_request_path(@user.id), class: 'friend_request' %>
+        </button>
+      <% else %>
+        <%= nil %>
+      <% end %>
```

Comment:
That's, `if/else`-`if/elsif/else` statement, what we call **Logic into Views**.
Please, use **View Helpers** in such a situation.